pytorch_lightning/plugins/training_type/ddp.py

```diff
@@ -177,7 +180,19 @@ def set_world_ranks(self):
         self.global_rank = self.node_rank * self.num_processes + self.local_rank
         self.world_size = self.num_nodes * self.num_processes
 
+    def pre_configure_ddp(self):
+        # todo: PyTorch 1.7.0 DDP introduces ``self.reducer._rebuild_buckets()``` breaking manual_optimization
+        if _TORCH_GREATER_EQUAL_1_7 and not self.lightning_module.automatic_optimization:
+            rank_zero_warn(
+                "From PyTorch 1.7.0, Lightning ``manual_optimization`` needs to set ``find_unused_parameters=True`` "
+                "to properly work with DDP."
+            )
```

Comment:
Can you check if it's set to True already before warning?

Reply:
Addressed in 23a9a10, is it what you had in mind?